content/en/monitors/create/types/ci.md

```diff
@@ -19,18 +20,25 @@ further_reading:
 <div class="alert alert-warning">CI Visibility is not available in the selected site ({{< region-param key="dd_site_name" >}}) at this time.</div>
 {{< /site-region >}}
 
-<div class="alert alert-info">CI Pipeline monitors are in alpha.
+<div class="alert alert-info">CI monitors are in alpha.
 </div>
 
 ## Overview
 
-Once [CI Visibility is enabled][1] for your organization, you can create a CI Pipeline monitor to alert you when a specified type of CI Pipeline event exceeds a user-defined threshold over a given period of time.
+Once [CI Visibility is enabled][1] for your organization, you can create a CI Pipeline or Test monitor.
+
+CI monitors allow you to visualize CI data and set up alerts on it. For example, create a CI Pipeline monitor to receive alerts on a failed pipeline or a job. Create a CI Test monitor to receive alerts on failed or slow tests.
 
 ## Monitor creation
 
-To create a [CI Pipeline monitor][2] in Datadog, use the main navigation: **Monitors -> New Monitor --> CI Pipelines**.
+To create a [CI monitor][2] in Datadog, use the main navigation: *Monitors -> New Monitor --> CI*.
+
+<div class="alert alert-info"><strong>Note</strong>: There is a default limit of 1000 CI monitors per account. <a href="/help/">Contact Support</a> to lift this limit for your account.</div>
+
+Choose between a **Pipelines** or **Tests** monitor:
```

Comment:
Agreed, this line needs to be removed, and the action statement should go at the beginning of the tab text in each section.

```suggestion
```